app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+  include ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
```

Comment:
@blomquisg I'm not against doing this I just don't know how :)  I include the mixin in event_catcher_spec as well as in the runner.  I tried various ways to then require runner.rb in the spec but couldn't get it to work. I need some ruby help on this one...